### PR TITLE
Allow the `salt-call` command execution to be bypassed

### PR DESCRIPTION
**Pull request by s0undt3ch**

### CODE REVIEW

File: lib/kitchen/provisioner/salt_solo.rb [hunk 1]
```diff
@@ -284,6 +285,8 @@ def run_command
         debug("running driver #{name}")
         debug(diagnose)
 
+        return unless config[:run_salt_call]
```
Comment (gtmanfred) [inline lib/kitchen/provisioner/salt_solo.rb:288]: Can you add this to the documentation pages?

**Review by gtmanfred — CHANGES_REQUESTED** [review]

**Comment by s0undt3ch** [comment]
There. Is that enough?

**Review by gtmanfred — APPROVED** [review]
perfect!

**Comment by daks** [comment]
just for my knowledge, what is the use case of not running salt-call inside a Kitchen VM?

I think Kitchen can be configured to not have a provisioner, if you only want to kitchen create and kitchen verify. _(but I may be wrong)_

**Comment by s0undt3ch** [comment]
We're in the process of creating "golden" images, which means, we previously ran the salt states against the VM, which makes this step unnecessary.

**Comment by gtmanfred** [comment]
There are still other stuff in the kitchen-salt stuff that is being used (specifically to transport the salt repo over to the testing instance).

**Comment by gtmanfred** [comment]
v0.5.0 released with your recent additions @s0undt3ch

**Comment by s0undt3ch** [comment]
Thanks!

**Comment by daks** [comment]
@s0undt3ch We use golden images too, but we create Vagrant images and don't use these images with Kitchen.

**Pull request merged**